File: src/main/java/itmo/is/project/mapper/module/dock/DockingSpotMapper.java
package itmo.is.project.mapper.module.dock;

import itmo.is.project.dto.module.dock.DockingSpotDto;
import itmo.is.project.mapper.EntityMapper;
import itmo.is.project.mapper.user.SpaceshipMapper;
import itmo.is.project.model.module.dock.DockingSpot;
import org.mapstruct.Mapper;

@Mapper(componentModel = "spring", uses = {SpaceshipMapper.class})
public interface DockingSpotMapper extends EntityMapper<DockingSpotDto, DockingSpot> {
}

File: src/main/kotlin/itmo/isproject/mapper/module/dock/DockingSpotMapper.kt
package itmo.isproject.mapper.module.dock

import itmo.isproject.dto.module.dock.DockingSpotDto
import itmo.isproject.mapper.EntityMapper
import itmo.isproject.mapper.user.SpaceshipMapper
import itmo.isproject.model.module.dock.DockingSpot
import org.mapstruct.Mapper

@Mapper(uses = [SpaceshipMapper::class])
interface DockingSpotMapper : EntityMapper<DockingSpotDto, DockingSpot>
